moped-editor/src/views/projects/projectsListView/ProjectsListViewFiltersConf.js

```diff
@@ -265,17 +265,26 @@ export const PROJECT_LIST_VIEW_FILTERS_CONFIG = {
       lookup: {
         table_name: "moped_fund_sources",
         getOptionLabel: (option) => option.funding_source_name,
-        operators: [
-          "string_equals_case_insensitive",
-          "string_does_not_equal_case_insensitive",
-        ],
+        operators: ["string_contains_case_insensitive"],
+      },
+      operators: [
+        "string_contains_case_insensitive",
+        "string_is_null",
+        "string_is_not_null",
+      ],
+    },
+    {
+      name: "funding_source_and_program_names",
+      label: "Funding program",
+      placeholder: "Funding program",
+      type: "string",
+      lookup: {
+        table_name: "moped_fund_programs",
+        getOptionLabel: (option) => option.funding_program_name,
+        operators: ["string_contains_case_insensitive"],
       },
       operators: [
         "string_contains_case_insensitive",
-        "string_begins_with_case_insensitive",
-        "string_ends_with_case_insensitive",
-        "string_equals_case_insensitive",
-        "string_does_not_equal_case_insensitive",
```

Comment:
Thanks for removing these! I definitely think we should keep this change in this PR since it is a search on a concatenated list.